packages/koa-liveness-ping/README.md

```diff
@@ -0,0 +1,30 @@
+# `@shopify/koa-liveness-ping`
+
+A package for creating a liveness ping middleware for use with koa
```

Comment:
What do you think about this instead? `A koa middleware package for creating a liveness ping.`

Reply:
I think we need a consistent voice and tone on these descriptions. I like the following in this case, actually:

> Create a liveness ping koa middleware

thoughts?

Reply:
Sounds good to me!